tools/osbuilder/node-builder/azure-linux/README.md

```diff
@@ -0,0 +1,285 @@
+
+If you intend to build the confpods UVM, install the following package:
+```sudo dnf install -y kernel-uvm-devel``
+
+## Optional: Build and deploy the containerd fork from scratch
+
+```
+git clone --depth 1 --branch tardev-v1.7.7 https://github.com/microsoft/confidential-containers-containerd.git
+pushd confidential-containers-containerd/
+GODEBUG=1 make
+popd
+```
+
+Overwrite existing containerd binary, restart service:
+```
+sudo cp -a -S .bak -b confidential-containers-containerd/bin/containerd /usr/bin/containerd
+sudo systemctl restart containerd
+```
+
+# Use Kata tooling to build the Kata host and guest components
+
+If these instructions are not being followed based on having cloned its hosting repository onto the system, clone the kata-containers repository:
+```git clone --depth 1 --branch mahuber/reproducible-builds https://github.com/microsoft/kata-containers.git```
+
+To build Azure Linux's kata-containers package components and UVM, run:
+```
+pushd kata-containers/tools/osbuilder/node-builder/azure-linux
+make all
+popd
+```
+To build the kata-containers-cc package components and UVM, run `make all-confpods`.
+The `all` target runs the `clean[-confpods]`, `package[-confpods]` and `uvm[-confpods]` targets in a single step (the `uvm` target depends on the `package` target).
+
+**Note:** By default, a UVM with Kata Agent enforcing a restrictive policy will be built, this requiring to generate valid pod security policy annotations using the `genpolicy` tool. While always recommended to do keep doing so, a UVM with a permissive security policy can be built by adapting the file `tools/osbuilder/node-builder/azure-linux/uvm_build.sh` and changing the `AGENT_POLICY_FILE` variable assignment to `allow-all.rego`
+
+# Install the built components
+
+In this step, we move the build artifacts to proper places and eventually restart containerd so that the new Kata(-CC) configuration files are loaded.
+The following commands refer to having built the components and UVM for Azure Linux's kata-containers package in the prior step.
+If you built the kata-containers-cc package components, call the `deploy-confpods` make target.
+
+```
+pushd kata-containers/tools/osbuilder/node-builder/azure-linux
+sudo make deploy
+popd
+```
+
+# Run Kata (Confidential) Containers
+
+## Run via CRI or via containerd API
+
+Use e.g. `crictl` (or `ctr`) to schedule Kata(-CC) containers, referencing either the Kata or Kata-CC handlers.
+
+The following sets of commands serve as a general reference for installing `crictl` and setting up some basic CNI to run pods:
+- Install `crictl`, set runtime endpoint in `crictl` configuration:
+
+  `sudo dnf -y install cri-tools`
+
+  `sudo crictl config --set runtime-endpoint=unix:///run/containerd/containerd.sock`
+
+- Install CNI binaries and set a sample CNI config:
+
+  `sudo dnf -y install cni`
```

Comment:
Is there some "official" CNI doc that we can point to, instead of including these setup steps here?

Reply:
Addressed as discussed